Btu-building-philadelphia/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html>
```

Comment:
**accessibility suggestion:** Your `html` element should always have a `lang` attribute.
```suggestion
<html lang="en">
```